test/sample-configuration-files/client.crt

```diff
@@ -0,0 +1,103 @@
+Certificate:
+    Data:
+        Version: 3 (0x2)
```

Comment:
Hmm, surprised the parser ignores this when not prefixed by `#` ??

Reply:
X509.Certificate/Private_key.decode_pem only looks for `-----BEGINYYYY------ ... -----ENDYYYY-----` lines :)

which in my opinion is pretty fine (from looking into my global trust anchors (/etc/ssl/cert.pem), this is definitively the way to go)